site/_docs/installation.md

```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
 simply run the following command to install Jekyll:
 
 {% highlight shell %}
-$ gem install jekyll
+$ gem install bundler
 {% endhighlight %}
 
+Now you can install Jekyll simply by running `bundle install`.
```

Comment:
For the `jekyll new` command to run, jekyll has to be installed in the system.
So for first time users, `gem install jekyll` is the way.

bundle install will install the minima gem or any theme gem